packages/twenty-server/src/modules/workflow/workflow-executor/workspace-services/workflow-executor.workspace-service.ts

```diff
@@ -44,26 +48,49 @@ export class WorkflowExecutorWorkspaceService {
       payload,
     });
 
+    const stepOutput = {
+      type: step.type,
+      result: result.result,
+      error: result.error,
```

Comment:
I think we should be careful when storing the raw error. Indeed, here, we store the error with the stack trace, showing the internal file system of the worker running the jobs. These errors will be displayed to the user, and it's usually considered unsafe to expose stack traces publicly.